src/Rendering/VTKJS/vtk/ItkVtkViewProxy.js

```diff
@@ -702,6 +704,16 @@ function ItkVtkViewProxy(publicAPI, model) {
   model.axesCircleRadius = 4
   model.axesTextOffset = 14
 
+  model.enableLabelBBox = false
```

Comment:
Is it possible to avoid duplication of state and pull this boolean from the context?

Reply:
I am using the model's enabled flag to [guard against](https://github.com/Kitware/itk-vtk-viewer/pull/699/files#diff-ea69b57bef01cbc2983c1fd92fdc8bce80a470cda143616cb12b46643f6c1a71R953-R963) unnecessary render calls by comparing it to the context state.

Reply:
Could pull the visibility boolean from the actor. Or just call `actor.setVisiblity` and if vtk.js is working right, it will do the guard check and won't render if nothing changed.

Reply:
Cleaned up :+1: